x/storage/keeper/keeper.go

```diff
@@ -567,6 +559,12 @@ func (k Keeper) CreateObject(
 		return sdkmath.ZeroUint(), err
 	}
 
+	// check object
+	_, found = k.GetObjectInfo(ctx, bucketName, objectName)
```

Comment:
This check is already implemented below.

Reply:
fixed